Makefile

```diff
@@ -118,14 +118,14 @@ DEPENDABOT_PATH=".github/dependabot.yml"
 .PHONY: gendependabot
 gendependabot:
 	@echo "Recreate dependabot.yml file"
-	@echo -e "# File generated by \"make gendependabot\"; DO NOT EDIT.\n" > ${DEPENDABOT_PATH}
```

Comment:
this was for ubuntu users. I think the right solution is to avoid using "\n", so this will become:
```
@echo "# File generated by \"make gendependabot\"; DO NOT EDIT." > ${DEPENDABOT_PATH}
@echo  "" >> ${DEPENDABOT_PATH}
@echo "version: 2" >> ${DEPENDABOT_PATH}
@echo "updates:" >> ${DEPENDABOT_PATH}
@echo "Add entry for \"/\""
@echo "  - package-ecosystem: \"gomod\"" >> ${DEPENDABOT_PATH}
@echo "     directory: \"/\"" >> ${DEPENDABOT_PATH}
@echo "     schedule:" >> ${DEPENDABOT_PATH}
@echo "       interval: \"weekly\"" >> ${DEPENDABOT_PATH}
...
```

Reply:
Works on both now.